Dockerfile

```
@@ -1,4 +1,4 @@
-FROM golang:latest AS build
+FROM golang:1.17.1-bullseye AS build
```

```suggestion
FROM golang:1.17-bullseye AS build
```

How about only tagging minor so we automatically get security/patch fixes?

---

would be an idea, but we would not be able anymore to rebuild old containers as they are. Reproducible builds.

---

Maybe add something like depenendbot or something like that.

---

Dependabot or renovate would make sense regardless.


I am not sure how go and go.mod works to the detail of reproducible builds?

---

As far as I know are the dependencies  in go.mod have versions and they are not automatically updated. Also with the tags in the repo. The version of the dependency of the tag would be used.